### ISSUE TITLE
[Suggestion] Option to disable of each tier

### ISSUE THREAD

**Issue by Samtrion** [p455w0rd/DankNull#78]
It would be nice, if we could decide which /dank/null tier is available.

For example only the diamond/emerald tier

> B:EnableRedstoneTier=false
> B:EnableLapisTier=false
> B:EnableIronTier=false
> B:EnableGoldTier=false
> B:EnableDiamondTier=true
> B:EnableEmeraldTier=true

These options not only deactivate the recipes, they also ensure that the respective Tier is not registered.

**Comment by p455w0rd** [comment]
no...this would cause confusion with the metadata used for each tier

**Comment by p455w0rd** [comment]
After a short amount of though, I will add this. This will just delay the next update a bit. in 1.13  meta data will be gone, therefore I can get a little ahead of the game now and prepare for that..this would allow for such a config to exist.

**Comment by p455w0rd** [comment]
this is possible via other mods in 1.12 and I just dont have the motivation or time to implement such a feature
